hack/make/.binary-setup

```diff
@@ -5,6 +5,7 @@ DOCKER_RUNC_BINARY_NAME='runc'
 DOCKER_CONTAINERD_BINARY_NAME='containerd'
 DOCKER_CONTAINERD_CTR_BINARY_NAME='ctr'
 DOCKER_CONTAINERD_SHIM_BINARY_NAME='containerd-shim'
+DOCKER_CONTAINERD_SHIM_RUNC_V2_BINARY_NAME='containerd-shim-runc-v2'
```

Comment:
We should consider getting rid of these env-vars; I don't think they're really used

Reply:
done